modules/cnf-about-the-profile-creator-tool.adoc

```diff
@@ -6,11 +6,21 @@
 [id="cnf-about-the-profile-creator-tool_{context}"]
 = About the Performance Profile Creator
 
-The Performance Profile Creator (PPC) is a command-line tool, delivered with the Node Tuning Operator, used to create the performance profile.
-The tool consumes `must-gather` data from the cluster and several user-supplied profile arguments. The PPC generates a performance profile that is appropriate for your hardware and topology.
+The Performance Profile Creator (PPC) is a command-line tool, delivered with the Node Tuning Operator, which can help you to create a performance profile for your cluster.
 
-The tool is run by one of the following methods:
+PPC discovery mode::
+Use the PPC tool in discovery mode to inspect your cluster. The PPC tool learns about your cluster in discovery mode by using the output from your `must-gather` data. 
```

Comment:
I think this might confuse people who still remember cnf-tests. We used to have a discovery mode there where the tool connected to the cluster and pulled the data it needed.

PPC will not pull the data. The user has to prepare must gather in advance.

Reply:
Updated to clarify that the PPC in discovery mode uses the must-gather data. 

I am still slightly unclear about the purpose of discovery mode though - the result of running discovery mode is output to the terminal telling the user certain details about their cluster that it scraped from the must-gather? 
Discovery mode does not retain this information to inform the PPC when creating the proposed performance profile?

Reply:
I must be missing something, because PPC has only one mode of operation. Must gather. And it is an one shot "filter" tool, it does not store anything.

Reply:
I've updated the docs to make this clearer. Perhaps the term "discovery mode" is still misleading, but hopefully the content explains it better now.